### PR TITLE
fix: update incorrect core color token values

### PR DESCRIPTION
**Pull request by pixelflips**
# Description

Some token hex values did not match the design specs
This adjusts the core values to match.

Fixes https://kajabi.atlassian.net/browse/DSS-1163

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests you've added and run to verify your changes.
Provide instructions so that we can reproduce.
Please also list any relevant details for your test configuration.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

### CODE REVIEW

File: libs/core/src/global/styles/tokens/base/core.json
Comment (monicawheeler) [inline libs/core/src/global/styles/tokens/base/core.json:1]: Hey Phillip, thanks for taking this on! I did a comparison between the original document provided by Design and the updated colors in this PR. Everything aligns but two updates are missing: 
* blue-150 should be #dbeafe
* blue-900 should be #1e3a8a

Google sheet with comparison:
https://docs.google.com/spreadsheets/d/1lMlGiboL3fNc3ZNDb-CtzXiwfsbDN_FIsZOtjvV87Jo/edit?usp=sharing

Comment (pixelflips) [reply libs/core/src/global/styles/tokens/base/core.json:1]: @monicawheeler great catch! thanks for the thorough review! Will update now.

**Comment by ju-Skinner** [comment]
@pixelflips I added another commit, which generated the `core` and `semantic` styles. I did this by running `npx nx run @pine-ds/core:build.tokens`

**Review by ju-Skinner — APPROVED** [review]

**Comment by pixelflips** [comment]
> @pixelflips I added another commit, which generated the `core` and `semantic` styles. I did this by running `npx nx run @pine-ds/core:build.tokens`

@ju-Skinner thank you!

**Review by monicawheeler — APPROVED** [review]
🙌🏻

**Review by anechol — APPROVED** [review]
Looks good!

**Pull request merged**